frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableColumnList/TableColumn/TableColumn.tsx

```diff
@@ -35,26 +35,34 @@ export const TableColumn: FC<TableColumnProps> = ({
           className={styles.primaryKeyIcon}
           role="img"
           aria-label="Primary Key"
+          strokeWidth={1.5}
```

Comment:
I've modified the ``strokeWidth`` to fit the design.

![ss 2550](https://github.com/user-attachments/assets/c169f1dc-04c1-4054-a575-fc399b7fa9b2)